build.sh

```diff
@@ -28,12 +28,12 @@ STACK_IMAGE="${STACK_IMAGE:-openliberty/application-stack:0.4}"
 #
 # URL at which your outer loop Dockerfile is hosted
 #
-DEVFILE_DOCKERFILE_LOC="${DEVFILE_DOCKERFILE_LOC:-https://github.com/OpenLiberty/application-stack/releases/download/outer-loop-0.4.0/Dockerfile}"
+DEVFILE_DOCKERFILE_LOC="${DEVFILE_DOCKERFILE_LOC:-https://github.com/OpenLiberty/application-stack/releases/download/outer-loop-0.4.1/Dockerfile}"
```

Comment:
This seems more like a 0.5 change than a 0.4.1 ... let's talk more at our next mtg. though, since from the devfile perspective it seems more like a 0.4.1.